### PR TITLE
jquery-ui.min.js made available locally

### PR DESCRIPTION
**Pull request by clover-es**
Removed link to jquery.com domain, issuing jquery-ui.min.js file as local resource.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.2.3
\ No newline at end of file
+0.2.3-local-jquery-ui
```
Comment (adamcharnock) [inline VERSION]: Can we leave this at 0.2.3? I'll do a new release once we merge the PR, but changing this here will upset the release system.

File: CHANGES.txt [hunk 2]
```diff
@@ -4,7 +4,13 @@ This file will be added to as part of each release
...
 ----
 
-Version 0.2.3, Tue 19 Dec 2017
+Version 0.2.3-local-jquery-ui, Sun 04 Mar 2018
+===============================
+
+b566d7879b	jquery-ui.min.js made available locally (clover-es)
+
+
+Version 0.2.3-local-jquery-ui, Tue 19 Dec 2017
```
Comment (adamcharnock) [inline CHANGES.txt]: Same here, leave this out, the `seed` build system will add it in automatically.

File: VERSION (hunk 1 above)
Comment (clover-es) [reply VERSION]: Yep! Sorry about that, I committed that in my repository but it accidentally went to PR as well :/ My first PR :P

**Comment by adamcharnock** [comment]
I've merged this first commit as discussed in #18. As the other commit does not need merging I'll need to close this PR rather than merge it in its entirety. Rest assured though, the desired change is now merged.

File: VERSION (hunk 1 above)
Comment (adamcharnock) [reply VERSION]: No worries, and congratulations on your first pull request! It is very much appreciated :-)

**Pull request closed without merging**